src/constraint/outcome/CollisionFreeOutcome.cpp

```diff
@@ -0,0 +1,82 @@
+#include <sstream>
+#include <aikido/constraint/outcome/CollisionFreeOutcome.hpp>
```

Comment:
Super nit: In a source file, we include the associated header very first then others in order of STL headers, 3rd party library headers, and AIKIDO headers.